### ISSUE TITLE
Create the Problems Page + Dynamic Routes

### ISSUE THREAD

**Issue by sebastian-nunez** [upe-webdev-2/Web-Dev-II-Frontend#14]
# Problems Page
> The problems page should display a tabulated list of problems available to us from the backend

## For now...
Use the data found in ```db.json``` to populate the table.

## ```/problems``` Page
Each row of the table should have the following:
1. ID of the problem
2. Title of the problem
3. Difficulty of the problem

**Note:** Once a user clicks on a problem row, the user should be taken to ```problems/ID_OF_THE_PROBLEM```.

### ```problems/ID_OF_THE_PROBLEM``` Page
We have to create a dynamic route that will essentially render an instance of the ```playground``` page with the data acquired from ```problems/ID_OF_THE_PROBLEM``` backend route

In other words, we need a ```[problemId].tsx``` component that handles all the rendering. 

- Look into getServerSideProps and Dynamic Routes using Next.js

**Comment by sebastian-nunez** [comment]
> This is what I have so far. The basic layout/functionality is mostly there.

```/problems```
<img width="1402" alt="Screen Shot 2022-10-18 at 12 22 52 PM" src="https://user-images.githubusercontent.com/45319275/196487843-755bfe8a-1daa-4598-95d6-69cdd433d334.png">

- the problems get added automatically and updated when you change ```db.json``` (our temp backend)

```problems/ID_OF_A_PROBLEM```
<img width="1420" alt="Screen Shot 2022-10-18 at 12 23 12 PM" src="https://user-images.githubusercontent.com/45319275/196487901-a5b25814-d317-4ecf-9e0e-c70d6bd162d4.png">

- the individual page generation is dynamic. the page has been created from the [problemID].tsx file (a template).

**Comment by GabrielPedroza** [comment]
I pulled the branch and so far, it looks great!!! Don't worry about making the code as optimized as possible. If the site is slow in general, we will fix that later on but as of right now, you guys are doing amazing!

**Comment by GabrielPedroza** [comment]
Also, I love that font!!!
